AWSWindowsHelpers/public/CloudFormation/ConvertTo-AWSWindowsHelperCFNParameter.ps1

```diff
@@ -0,0 +1,15 @@
+function ConvertTo-AWSWindowsHelperCFNParameter
```

Comment:
I think this function has value.

Reply:
I've cut out the New CFN Parameter function. The parameter for usePreviousValue has been added to the convert to CFN parameter function, to ensure previous functionality of being able to set the usePreviousValue value is retained.